src/modules/commander/commander_params.c

```diff
@@ -907,3 +907,20 @@ PARAM_DEFINE_INT32(COM_ASPD_FS_ACT, 0);
  */
 PARAM_DEFINE_INT32(COM_ASPD_FS_DLY, 0);
 
+/**
+ * User Flight Profile
+ *
+ * Describes the intended use of the vehicle.
+ * Can be used by ground control software or log post processing.
+ * This param does not influence the behavior within the firmware. This means for example the control logic is independent of the setting of this param (but depends on other params).
+ *
+ * 0: Default
```

Comment:
```suggestion
 * @value 0 Default
```

Reply:
Thanks. Fixed in https://github.com/PX4/Firmware/pull/12187/commits/434aebd4fd394010a691d13b2174e6ae95928841